sotopia/cli/benchmark/benchmark.py

```diff
@@ -296,3 +342,68 @@ def benchmark(
         verbose=False,
         push_to_db=True,
     )
+
+
+@app.command()
+def benchmark_all(
```

Comment:
Can you merge these two into the benchmark function?

```python 
@app.command()
def benchmark(
    model: str,
    models: List[str],
    only_show_performance: bool
)
```

Reply:
Addressed! 
I changed `model` to `models` for a clearer naming.
Also there are two questions:
1. What should be our default model
2. For displaying in command line, can we use pandas to format them? (I think pandas is not required in Sotopia)